api/types/types.go

```diff
@@ -116,6 +116,11 @@ var ControllerPolicyRule = []rbacv1.PolicyRule{
 		APIGroups: []string{"route.openshift.io"},
 		Resources: []string{"routes"},
 	},
+	{
+		Verbs:     []string{"get", "list", "watch"},
+		APIGroups: []string{"apps.openshift.io"},
+		Resources: []string{"deploymentconfigs"},
+	},
```

Comment:
I believe we will also need to remove this in the getControllerRules() function in client/router_create.go if the deploymentconfig resource does not exist.

Reply:
will do, thanks!

Reply:
Updated in c784d16e45a67cb56662ec796fb245c4e9a26089